src/crypto/tls/handshake_server_tls13.go

```diff
@@ -22,6 +22,45 @@ import (
 // messages cause too much work in session ticket decryption attempts.
 const maxClientPSKIdentities = 5
 
+// EXP_EventServerHandshakeTimingInfo is carries intra-stack durations for
```

Comment:
It's odd that this is defined in `handshake_server_tls13.go` but `EXP_EventClientHandshakeTimingInfo` is defined in `handshake_client.go`.

Reply:
Yeah, that's leftover from including this in the 1.2 CH struct. I'll remove it from there, and then move this to the 1.3-specific file.